### ISSUE TITLE
Categorizing Notebooks for Jupyter{Book} chapters

### ISSUE THREAD

**Issue by lohithmunakala** [ioos/notebooks_demos#411]
@ocefpaf and @MathewBiddle, here is the list of notebooks sorted according to the date of publication/issue.

| notebook                                            | Chapter                         |
|-----------------------------------------------------|---------------------------------|
| 2016-10-12-fetching_data.ipynb                      | data_access                     |
| 2016-11-15-glider_data_example.ipynb                | data_access                     |
| 2016-11-16-CF-UGRID-SGRID-conventions.ipynb         | data_analysis_and_visualization |
| 2016-12-19-exploring_csw.ipynb                      | data_access                     |
| 2016-12-20-searching_glider_deployments.ipynb       | data_access                     |
| 2016-12-22-boston_light_swim.ipynb                  | data_access                     |
| 2017-01-18-siphon-explore-thredds.ipynb             | data_access                     |
| 2017-01-23-R-notebook.ipynb                         | data_analysis_and_visualization |
| 2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb             | data_access                     |
| 2017-03-30-octave_notebook_example.ipynb            | data_analysis_and_visualization |
| 2017-05-14-running_compliance_checker.ipynb         | data_management                 |
| 2017-06-12-NCEI_RA_archive_history.ipynb            | data_access                     |
| 2017-07-25-HFRadar_currents.ipynb                   | data_access                     |
| 2017-08-01-xtractoR.ipynb                           | data_access                     |
| 2017-09-09-hurricane_irma.ipynb                     | data_access                     |
| 2017-11-01-Creating-Archives-Using-Bagit.ipynb      | data_managment                  |
| 2017-11-30-rerddap.ipynb                            | data_access                     |
| 2017-12-15-finding_HFRadar_currents.ipynb           | data_access                     |
| 2018-02-20-obis.ipynb                               | data_managment                  |
| 2018-02-27-pocean-timeSeries-demo.ipynb             | data_management                 |
| 2018-03-01-erddapy.ipynb                            | data_access                     |
| 2018-03-15-ssh-skillscore.ipynb                     | data_analysis_and_visualization |
| 2018-03-30-wave_height_assessment.ipynb             | data_analysis_and_visualization |
| 2018-12-04-grids.ipynb                              | data_analysis_and_visualization |
| 2019-02-26-hurricane_gis_part01.ipynb               | data_access                     |
| 2019-03-08-grids-temperature.ipynb                  | data_access                     |
| 2019-05-27-hurricane_gis_part02.ipynb               | data_access                     |
| 2020-02-14-QARTOD_ioos_qc_Water-Level-Example.ipynb | data_analysis_and_visualization |
| 2020-10-10-GTS.ipynb                                | data_managment                  |
| 2020-12-08-DataToDwC.ipynb                          | data_managment                  |

**Comment by ocefpaf** [comment]
> @ocefpaf and @MathewBiddle, here is the list of notebooks sorted according to the date of publication/issue.

Let's edit in your own comment above with our opinions for the categories. I'll make it a table with Matt's and my take on it.

**Comment by MathewBiddle** [comment]
I only had one potential change. The running_compliance_checker notebook should probably be data_management.

**Comment by MathewBiddle** [comment]
`pocean-timeSeries-demo.ipynb` should probably be data_management too. It is more about aligning to data management best practices then showing how to analyze and visualize.

**Comment by MathewBiddle** [comment]
@mwengren For the Jupyter{book} we're looking at three chapters (with notebooks organized as described in the first comment):
1. Data Access
2. Data Analysis and Visualization
3. Data Management

If you have comments on the chapters or the organization please add them here.

**Comment by ocefpaf** [comment]
@lohithmunakala both edits from @MathewBiddle make sense and I'll consolidate the table above with them. Let's wait for @mwengren to close this issue and port these changes to #409

**Comment by lohithmunakala** [comment]
Sure.

**Comment by mwengren** [comment]
Looks good, I am fine with categories as is.  

We should consider renaming some notebooks either as part of the transition to JupyterBook or shortly afterward.  

For example, for some of the IOOS Catalog-related notebooks, I'd like to use the name 'IOOS Catalog' explicitly.  This comment is based on the notebook titles used in the existing Data Demo Center site (https://ioos.github.io/notebooks_demos/code_gallery/) where the Titles differ from the physical notebook names:

- How to Search the IOOS CSW catalog with Python tools (https://ioos.github.io/notebooks_demos/notebooks/2016-12-19-exploring_csw/) - change to 'Searching the IOOS Catalog CSW service in Python'
- Fetching Data from a CSW catalog with Python tools (https://ioos.github.io/notebooks_demos/notebooks/2017-12-15-finding_HFRadar_currents/) - change to 'Fetching Data from the IOOS Catalog CSW service in Python'

Not sure if the existing titles will carry over directly to JupyterBook or not, but this would be a good opportunity to improve on those at the same time.  IOOS Catalog is just an example, we might want to make other changes as well.

**Comment by ocefpaf** [comment]
Micah, let me open a new issue for the renaming so we can address that after #409 is merged. I would not change them now to avoid conflicts with @lohithmunakala's PR.

**Comment by lohithmunakala** [comment]
If the categories are fine, should I make these changes to the PR?

**Comment by lohithmunakala** [comment]
> Micah, let me open a new issue for the renaming so we can address that after #409 is merged. I would not change them now to avoid conflicts with @lohithmunakala's PR.

The headings take the name of the First Markdown Cell in the notebook. Changes should be fine and not cause conflict (until I add the `_build` files). I haven't pushed any code that has the `_build` files or the content repo - yet.

**Comment by ocefpaf** [comment]
> If the categories are fine, should I make these changes to the PR?

Yes please!

> @ocefpaf, The headings take the name of the First Markdown Cell in the notebook. Changes should be fine and not cause conflict (until I add the _build files). I haven't pushed any code that has the _build files or the content repo - yet.

It would cause problems with the moved files. Let's finish this one so we can all start working on the new directory structure.

**Comment by lohithmunakala** [comment]
Great! Closing this issue then.
